frontend/src/component/layout/LayoutPicker/LayoutPicker.tsx

```diff
@@ -1,37 +1,19 @@
+import { FC, ReactNode } from 'react';
 import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
-import { matchPath } from 'react-router';
-import { useLocation } from 'react-router-dom';
 import { MainLayout } from '../MainLayout/MainLayout';
-import { ReactNode } from 'react';
 
 interface ILayoutPickerProps {
     children: ReactNode;
+    isStandalone?: boolean;
 }
 
-export const LayoutPicker = ({ children }: ILayoutPickerProps) => {
-    const { pathname } = useLocation();
-
-    return (
-        <ConditionallyRender
-            condition={isStandalonePage(pathname)}
-            show={children}
-            elseShow={<MainLayout>{children}</MainLayout>}
-        />
-    );
-};
-
-const isStandalonePage = (pathname: string): boolean => {
-    return standalonePagePatterns.some(pattern => {
-        return matchPath(pattern, pathname);
-    });
-};
-
-const standalonePagePatterns = [
-    '/login',
-    '/new-user',
-    '/reset-password',
-    '/reset-password-success',
-    '/forgotten-password',
-    '/splash/:splashId',
-    '/404',
-];
+export const LayoutPicker: FC<ILayoutPickerProps> = ({
+    isStandalone,
+    children,
+}) => (
+    <ConditionallyRender
+        condition={isStandalone === true}
```

Comment:
Does this need `===` or it can be used straight away?

Reply:
can be `Boolean(isStandalone)`, because prop can be `undefined`